### PR TITLE
Note configuration file requirements

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

We've had a couple of comments that the required configuration
files are not as documented as they should be.  Adding a note to
the install and troubleshooting guides to hopefully help.

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -204,6 +204,10 @@ The build steps on Debian are otherwise the same as Ubuntu, above.
 
 ## Configuration files
 
+The following configuration files are required in order for Buildah to run appropriately.  The
+majority of these files are contained in the `containers-common` package, please verify that
```
Comment (rhatdan) [inline install.md]: Can you state that this is commonly packaged in containers-common.  No need to force all distros to package that way.

Comment (TomSweeneyRedHat) [reply install.md]: hmm, didn't get e-mail on this.  I've add commonly and have removed ', please verify...' as it tells them to check for that kit.

**Comment by TomSweeneyRedHat** [comment]
pushed latest with requested change.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan, but still doesn't like me, can you retest this please?

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**